include/samplers/hmc_heur_sam.h

```diff
@@ -0,0 +1,112 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
I propose to use a less cryptic name for the file i.e. sampler instead of sam, what is heur stands for?